Попытка 1

NickPYTH opened the pull request.

NickPYTH committed:
Авторизация и регистрация добавлены

NickPYTH committed:
Начал работу над чатами

NickPYTH committed:
Сделан список частов, просмотр профиля, страницы ошибок

NickPYTH committed:
Доработал страницу чата

NickPYTH committed:
Редактирование профиля и изменение пароля

NickPYTH committed:
Обновлен Readme

NickPYTH committed:
Добавил ссыклу на дизайн в фигме

NickPYTH committed:
Добавил перенос в конец файла, не проходили автотесты

NickPYTH committed:
Update style.scss

NickPYTH committed:
Update main.scss

NickPYTH committed:
Прошелся скриптом и добавил доп строки в конец файлов

NickPYTH committed:
Прошелся скриптом и добавил доп строки в конец файлов

mikebars1995 commented on src/styles/main.scss:
```diff
@@ -0,0 +1,211 @@
+}
+
+input:defined {
+    background-color: #ffffff;
```
**Можно лучше**

все **цвета** лучше вынести в `css`-переменные и использовать их, чтобы можно было мгновенно понимать, что это за цвет, и можно было бы подкорректировать его в одном месте  и не искать дублирования во всем коде

mikebars1995 commented on src/index.html:
```diff
@@ -0,0 +1,21 @@
+    <div class="page-wrapper">
+        <div class="links-list">
+            <h3>{{pageName}}</h3>
+            {{#each pageLinks}}
```
**Нужно исправить**

Нужно обернуть список с ссылками в тег `nav` по семантике, так как это навигация по сайту.

mikebars1995 commented on src/pages/login/login.html:
```diff
@@ -0,0 +1,43 @@
+</head>
+<body>
+    <div class="page-wrapper">
+        <div class="login-page">
```
**Нужно исправить**

По семантике нужно помещать весь главный контент страницы в тег `main`.

**Можно лучше**

Чтобы сделать сайт более доступным для пользователя, нужно стараться придерживаться **семантической верстки**.
Несколько полезных статей на эту тему:
•	[Что такое семантическая вёрстка и зачем она нужна](https://htmlacademy.ru/blog/html/semantics)
•	[Семантика для циников](https://www.youtube.com/watch?v=B5pomLr059U)

mikebars1995 commented on src/pages/registration/registration.html:
```diff
@@ -0,0 +1,79 @@
+                    {{> text-input 
+                        label="Имя" 
+                        id="firstName" 
+                        name="firstName" 
```
**Нужно исправить**


По заданию 1го спринта нужно давать инпутам **конкретные** атрибуты `name`, чтобы в 3м спринте не менять ничего при подключении к `Api`. Скрин из задания https://disk.yandex.ru/i/lSPqeGrAa20ayQ

mikebars1995 reviewed (commented):
Здравствуйте. (**Нужно развернуть общий комментарий ↓**)

Посмотрите на *гитхабе* все комментарии к коду (нужно прокрутить вниз страницу там)

Работа проделана огромная


Что сделано отлично:


* `Readme` хорошо оформлен
* Проект задеплоен и отлично работает
* Отлично, что не забыли про `.gitignore`
* Отлично, что ограничиваете версию `node`
* Отлично, что нет `EOF` ошибок в гите
* Хорошая структура папок и файлов

Что нужно исправить:

* Нужно обернуть список с ссылками в тег `nav` по семантике, так как это навигация по сайту.
* По семантике нужно помещать весь главный контент страницы в тег `main`.
* По заданию 1го спринта нужно давать инпутам **конкретные** атрибуты `name`, чтобы в 3м спринте не менять ничего при подключении к `Api`. Скрин из задания https://disk.yandex.ru/i/lSPqeGrAa20ayQ 

Что можно улучшить:

* все **цвета** лучше вынести в `css`-переменные и использовать их, чтобы можно было мгновенно понимать, что это за цвет, и можно было бы подкорректировать его в одном месте  и не искать дублирования во всем коде


Исправьте, пожалуйста, недочеты и работа будет принята. Пожалуйста, проверьте работоспособность проекта и наличие возможных ошибок в консоли браузера (кнопка `F12`) перед отправкой на ревью. 

Напоминаю, что работа может быть принята только после исправления всех критических замечаний `Нужно исправить`.

**Комментарии `Можно лучше` не обязательны к исправлению прямо сейчас, это рекомендации**

Удачного рефакторинга кода.

NickPYTH committed:
- Обернут список с ссылками в тег nav
- Весь главный контент страниц помещен в тег main
- Даны инпутам конкретные атрибуты "name"
- Цвета частично вынесены в переменные, остальное доделаю по ходу 2 спринта

NickPYTH committed:
newline

NickPYTH committed:
newline
